Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/HOKSAMLAssertionBuilder.java

```diff
@@ -380,33 +366,24 @@ protected Subject createSubject(final String x509Name, final X509Certificate cer
     public Evidence createEvidence(final CallbackProperties properties, final Subject subject) {
         LOG.debug("SamlCallbackHandler.createEvidence() -- Begin");
         final String evAssertionID = properties.getEvidenceID();
-        final DateTime issueInstant = properties.getEvidenceInstant();
-        final String format = properties.getEvidenceIssuerFormat();
-        final DateTime beginValidTime = properties.getEvidenceConditionNotBefore();
-        final DateTime endValidTime = properties.getEvidenceConditionNotAfter();
         final List<AttributeStatement> statements = createEvidenceStatements(properties);
-
         final String issuer = properties.getEvidenceIssuer();
-
-        return buildEvidence(evAssertionID, issueInstant, format, beginValidTime, endValidTime, issuer, statements,
-                subject);
+        return buildEvidence(new BuildEvidenceParameter(evAssertionID, issuer, properties, statements, subject));
     }
 
     /**
      *
-     * @param evAssertionID
-     * @param issueInstant
-     * @param format
-     * @param beginValidTime
-     * @param endValidTime
-     * @param issuer
-     * @param statements
-     * @param subject
+     * @param parameterObject
+     *
      * @return
      */
-    public Evidence buildEvidence(String evAssertionID, DateTime issueInstant, String format, DateTime beginValidTime,
-            DateTime endValidTime, final String issuer, final List<AttributeStatement> statements, final Subject subject) {
+    public Evidence buildEvidence(BuildEvidenceParameter parameterObject) {
 
+        DateTime endValidTime = parameterObject.getEndValidTime();
```

Comment:
Why are you creating these local variables just to pass to another method?  just send the parameterObject to the next method.